spring.mail.host=smtp.gmail.com
spring.mail.port=587
spring.mail.username=chibuisi.test.dev@gmail.com
spring.mail.password=dev.test
spring.mail.properties.mail.smtp.starttls.enable=true
spring.mail.properties.mail.smtp.starttls.required=true
spring.mail.properties.mail.smtp.auth=true
spring.mail.properties.mail.smtp.connectiontimeout=5000
spring.mail.properties.mail.smtp.timeout=5000
spring.mail.properties.mail.smtp.writetimeout=5000


#spring.datasource.url=jdbc:mysql://localhost:3306/loyaltycrm
spring.datasource.username=
spring.datasource.password=password
spring.jpa.hibernate.ddl-auto=update
spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5InnoDBDialect
spring.cloud.gcp.sql.database-name=loyaltycrm
spring.cloud.gcp.sql.instance-connection-name=ford-loyalty-experimentation:us-central1:loyalty-crm
# So app starts despite "table already exists" errors.
spring.datasource.continue-on-error=true
# Enforces database initialization
spring.datasource.initialization-mode=always

spring.profiles.active=mysql
